api/institutions/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from api.institutions import views
+
+urlpatterns = [
+    url(r'^$', views.InstitutionList.as_view(), name=views.InstitutionList.view_name),
+    url(r'^(?P<institution_id>\w+)/$', views.InstitutionDetail.as_view(), name=views.InstitutionDetail.view_name),
+    url(r'^(?P<institution_id>\w+)/nodes/$', views.InstitutionNodeList.as_view(), name=views.InstitutionNodeList.view_name),
+    url(r'^(?P<institution_id>\w+)/nodes/(?P<node_id>\w+)/$', views.InstitutionNodeDetail.as_view(), name=views.InstitutionNodeDetail.view_name),
+    url(r'^(?P<institution_id>\w+)/users/$', views.InstitutionUserList.as_view(), name=views.InstitutionUserList.view_name),
+    url(r'^(?P<institution_id>\w+)/users/(?P<user_id>\w+)/$', views.InstitutionUserDetail.as_view(), name=views.InstitutionUserDetail.view_name),
```

Comment:
Should `institutions/{instution_id}/users/{user_id}/` and `institutions/{institution_id}/nodes/{node_id}/` be a thing?  You can have multiple URL's pointing to the same resource, but you don't actually give users these links.  If you navigate to `/institution/institution_id/users/`, this endpoint is using the UserSerializer, so each of the self links in the list point to the UserDetail view (`users/user_id`), not `institutions/{instution_id}/users/{user_id}/`. 

I don't these two links are needed since they don't reveal anything that `/users/user_id` and `nodes/node_id` don't already reveal.